src/web/modules/services/performancePlatformCsv.ts

```diff
@@ -0,0 +1,50 @@
+import { Parser } from 'json2csv'
+import { Service } from '../../../lib/pay-request/types/adminUsers'
+import _ from 'lodash'
+import moment from 'moment'
+
+const fields = [
+  {
+    label: "_timestamp",
+    value: "went_live_date"
+  }, {
+    label: 'service',
+    value: 'alwaysdefault',
+    default: 'govuk-pay'
+  }, {
+    label: 'service_id',
+    value: 'id'
+  }, {
+    label: 'agency',
+    value: 'agency'
+  }, {
+    label: 'service_name',
+    value: 'service_name'
+  }, {
+    label: 'count',
+    value: 'count',
+    default: '1'
+  }, {
+    label: 'sorting',
+    value: 'sorting'
+  }
+]
+
+export function format(liveServices: Service[]): string {
+  const parser = new Parser({ fields })
+  const sortedServices = _.sortBy(liveServices, 'merchant_details.name', 'service_name.en', 'id')
+  const csvData: any = []
+  let sorting = 1
+  sortedServices
+    .forEach(service => {
+      const row = {
+        went_live_date: moment(service.went_live_date).format('YYYY-MM-DD'),
+        id: service.id,
+        agency: service.merchant_details && service.merchant_details.name && service.merchant_details.name.trim() || '',
+        service_name: service.service_name.en.trim(),
+        sorting: sorting++
```

Comment:
`.forEach` gives you this incremented number for free by passing the index as a second param (although it will be 0 indexed)

something like: 
```js
.forEach((service, index) => csvData.push({
  //...
  sorting: index + 1
}))
```

Also it looks like this idea of one set -> format values -> push new value into another set could be cleanly described as a map (which gives index in the same way!)
```js 
const csvData = sortedServices.map((service, index) => ({
  // ...
}))
```

Reply:
Nice suggestions, I'll make these changes

Reply:
made those changes